src/vs/workbench/services/positronLayout/browser/positronCustomViews.ts

```diff
@@ -16,17 +18,22 @@ import './layouts/maximizedPartLayouts.js';
 import './layouts/positronHelpPaneDocked.js';
 
 
-type LayoutPick = IQuickPickItem & { layoutDescriptor: CustomPositronLayoutDescription };
+type LayoutPick = IQuickPickItem & {
+	layoutDescriptor: CustomPositronLayoutDescription;
+	precondition?: ContextKeyExpression;
+};
 
 export const positronCustomLayoutOptions: LayoutPick[] = [
 	positronFourPaneDsLayout,
 	positronTwoPaneLayout,
-	positronNotebookLayout
+	positronNotebookLayout,
+	positronAssistantLayout
 ].map(function positronLayoutInfoToQuickPick(layoutInfo: PositronLayoutInfo): LayoutPick {
 	return {
 		id: layoutInfo.id,
 		label: `$(${layoutInfo.codicon}) ${layoutInfo.label.value}`,
 		layoutDescriptor: layoutInfo.layoutDescriptor,
+		precondition: layoutInfo.precondition,
```

Comment:
```suggestion
		precondition: layoutInfo.precondition ?? ContextKeyExpr.true(),
```

Will need imports for ContextKeyExpr added above